src/Interactive/EditorFeatures/Core/Implementation/Interactive/Commands/HelpCommand.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Interactive;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.InteractiveWindow.Commands;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+{
+    [Export(typeof(IInteractiveWindowCommand))]
+    [InteractiveWindowRole(InteractiveWindowRoles.Any)]
+    internal sealed class HelpReplCommand : InteractiveWindowCommand
+    {
+        internal const string CommandName = "help";
+
+        private static readonly string[] s_details = new[]
+        {
+            "  command-name    Name of the REPL command to display help on.",
+        };
+
+        public override string Description
+        {
+            get { return "Display help on specified REPL command, or all available REPL commands and key bindings if none specified."; }
+        }
+
+        public override IEnumerable<string> Names
+        {
+            get { yield return CommandName; }
+        }
+
+        public override string CommandLine
+        {
+            get { return "[command-name]"; }
+        }
+
+        public override IEnumerable<string> DetailedDescription
+        {
+            get { return base.DetailedDescription; }
```

Comment:
Why is REPL used in the help command. I understand using abbreviations on this thread but they shouldn't be used in Help documents. Even for this thread a list somewhere would be helpful. I shouldn't have to use Google to look up words in a help file.

Sent from my iPhone
I apologize for any typos Siri might have made.
(503) 803-6077

> On Jul 15, 2015, at 1:52 PM, Andrew Casey <notifications@github.com> wrote:
> 
> REPL